test/library/standard/CommDiagnostics/commDiagsTable.comm-none.good

```diff
@@ -18,23 +18,23 @@
 | -----: |
 |      0 |
 
-| locale | get | get_nb | put | put_nb | test_nb | wait_nb | try_nb |      amo | execute_on | execute_on_fast | execute_on_nb | cache_get_hits | cache_get_misses | cache_put_hits | cache_put_misses |
-| -----: | --: | -----: | --: | -----: | ------: | ------: | -----: | -------: | ---------: | --------------: | ------------: | -------------: | ---------------: | -------------: | ---------------: |
-|      0 |   0 |      0 |   0 |      0 |       0 |       0 |      0 | unstable |          0 |               0 |             0 |              0 |                0 |              0 |                0 |
+| locale | get | get_nb | put | put_nb | test_nb | wait_nb | try_nb |      amo | execute_on | execute_on_fast | execute_on_nb | cache_get_hits | cache_get_misses | cache_put_hits | cache_put_misses | cache_num_prefetches | cache_num_page_readaheads | cache_prefetch_unused | cache_prefetch_waited | cache_readahead_unused | cache_readahead_waited |
+| -----: | --: | -----: | --: | -----: | ------: | ------: | -----: | -------: | ---------: | --------------: | ------------: | -------------: | ---------------: | -------------: | ---------------: | ------------------: | ------------------: | ------------------: | ------------------: | ------------------: | ------------------: |
+|      0 |   0 |      0 |   0 |      0 |       0 |       0 |      0 | unstable |          0 |               0 |             0 |              0 |                0 |              0 |                0 |                    0 |                         0 |                     0 |                     0 |                      0 |                      0 |
```

Comment:
I think the table is getting too noisy for a typical user. I wonder if we should make these new stats opt-in (and also wonder if we should lump the existing `cache_{get,put}_{miss,hit}` under this opt-in behavior.

Reply:
@ronawho - I don't remember all the details but I thought that by default we weren't including columns for things that are 0 all through. Does that help or is it still a concern to add these new fields because the cache is on by default?

Reply:
@ronawho is that something you would ask @thomasrolinger to do in this PR? Arguably it could/should be in a follow-up PR, maybe even by somebody else.

Reply:
A followup PR is fine with me.

Reply:
@thomasrolinger - would you note this in a Future Work section at the end of your PR message (the top of the PR page)? Thanks

Reply:
Done; cleaned up the PR message. Let me know if there is anything else you want to see.